integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -0,0 +1,290 @@
+// Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at
+// http://oss.oracle.com/licenses/upl.
+package oracle.kubernetes.operator;
+
+import static oracle.kubernetes.operator.BaseTest.DOMAININIMAGE_WLST_YAML;
+import static oracle.kubernetes.operator.BaseTest.OPERATOR1_YAML;
+import static oracle.kubernetes.operator.BaseTest.QUICKTEST;
+import static oracle.kubernetes.operator.BaseTest.logger;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import oracle.kubernetes.operator.utils.Domain;
+import oracle.kubernetes.operator.utils.ExecResult;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.TestUtils;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assume;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+/** Operator upgrade JUnit test file testing the operator upgrade from older releases to develop. */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class ItOperatorUpgrade extends BaseTest {
+
+  private static final String OP_BASE_REL = "2.0";
+  private static final String OP_TARGET_RELEASE = "weblogic-kubernetes-operator:latest";
+  private static final String DOM_TARGET_RELEASE_VERSION = "weblogic.oracle/v4";
```

Comment:
you can take this value from env variable also, so that it can be passed from jenkins or command line.

Reply:
Done.